include/bzfsAPI.h

```diff
@@ -45,7 +45,7 @@
 
 class bz_Plugin;
 
-#define BZ_API_VERSION	26
+#define BZ_API_VERSION	27
```

Comment:
after confirmation, no version bump is needed. it's only necessary when existing constructors, objects, or functions are modified